Fabric.Identity.API/scripts/Install-Identity-Discovery-IdPSS.ps1

```diff
@@ -15,6 +15,7 @@ param(
         return $true
     })] 
     [string] $installConfigPath = "$PSScriptRoot\install.config", 
+    [string] $azureConfigPath = "C:\Program Files\Health Catalyst\azuresettings.config",    # TODO: env variable for program files?
```

Comment:
Forgot to look into an environment variable or similar for the program files path, will try to find something

Reply:
According to Mr. Tabet, it's $env:ProgramFiles

Reply:
According to Mr. Tabet, it's $env:ProgramFiles